### PR TITLE
Has been created TDD for Instruction Entity

### PR DESCRIPTION
**Pull request by thivalls**

### CODE REVIEW

File: module/Application/src/Application/Entity/Interaction.php [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    public function setDatePosted($date_posted)
+    {
+        $this->date_posted = $date_posted;
+        return $this;
+    }
+
+    /**
+     * get description
+     *
+     * @return mixed
```
Comment (juizmill) [inline module/Application/src/Application/Entity/Interaction.php]: string --> mixed

File: module/Application/src/Application/Entity/Interaction.php [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    /**
+     * to array
+     *
+     * @return array Return array list
+     */
+    public function toArray()
+    {
+        return (new ClassMethods())->extract($this);
+    }
+
```
Comment (juizmill) [inline module/Application/src/Application/Entity/Interaction.php]: remove the two spaces

File: module/Application/src/Application/Entity/Interaction.php [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+}
```
Comment (juizmill) [inline module/Application/src/Application/Entity/Interaction.php]: add one space last line

File: module/Application/tests/ApplicationTest/Entity/InteractionTest.php [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        $class = new Interaction();
+        $class->setId('hello');
+
+        $class2 = new Interaction();
+        $class2->setId(-1);
+
+        $class3 = new Interaction();
+        $class3->setId(0);
+    }
+}
```
Comment (juizmill) [inline module/Application/tests/ApplicationTest/Entity/InteractionTest.php:105]: add one space last line

Comment (thivalls) [inline module/Application/tests/ApplicationTest/Entity/InteractionTest.php:105]: all refactored :stuck_out_tongue_winking_eye:

**Pull request merged**